scripts/zkai_attention_kv_fused_softmax_table_route_matrix_gate.py

```
@@ -0,0 +1,760 @@
+            mutation_results.append({"name": name, "rejected": False, "error": "mutation accepted"})
+    if tuple(item["name"] for item in mutation_results) != EXPECTED_MUTATION_NAMES:
+        raise FusedSoftmaxTableRouteMatrixGateError("mutation name/order drift")
+    result = copy.deepcopy(base)
+    result["mutation_results"] = mutation_results
+    result["mutations_checked"] = len(mutation_results)
+    result["mutations_rejected"] = sum(1 for item in mutation_results if item["rejected"] is True)
+    validate_result(result)
+    return result
+
+
+def validate_route_rows(rows: Any) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (20 > 12)

(PLR0912)